### PR TITLE
fix: Fix introductory macro examples

### PR DESCRIPTION
**Pull request by adigitoleo**
Current examples don't compile.

### CODE REVIEW

File: doc/tut3.rst [hunk 1]
```diff
@@ -188,21 +188,29 @@ Backticks are used to insert code from `NimNode` symbols into the
 generated expression.
 
 .. code-block:: nim
-    macro a(i) = quote do: let `i` = 0
+    macro a(i) = quote do:
+        let `i` = 0
```
Comment (ringabout) [inline doc/tut3.rst]: Please use two spaces indentation.

**Comment by ringabout** [comment]
You can also specify `:test: "nim c $1"` to make sure it compiles (use `nim rst2html doc/tut3.rst` to check the compilation)

see also https://nim-lang.github.io/Nim/contributing.html

**Comment by adigitoleo** [comment]
Failure looks unrelated, I'm not very experienced with CI though.

**Comment by ringabout** [comment]
CI failures should be unrelated.

You can easilly rerun failed CI following instructions here => https://nim-lang.github.io/Nim/contributing.html#the-cmdgit-stuff-debugging-ci-failures-flaky-tests-etc

> Azure: if on your own fork, it's possible from inside azure console (e.g. dev.azure.com/username/username/_build/results?buildId=1430&view=results) via rerun failed jobs on top. If either on you own fork or in Nim repo, it's possible from inside GitHub UI under checks tab, see https://github.com/timotheecour/Nim/issues/211#issuecomment-629751569

**Review by ringabout — APPROVED** [review]
LGTM

**Comment by adigitoleo** [comment]
Cheers

**Pull request merged**